### DISCUSSION TITLE
the precise significance depending on the location within the namelist of identical input parameters

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#12909] [Q&A]
Hello, I'm studying FDS, but I'm still confused even after reading the user guide.. and I have some questions.

1. Regarding the heat of combustion, it can indeed be specified on either the REAC or MATL line. The HEAT_OF_COMBUSTION (HOC) on the REAC line typically represents the heat release associated with the predominant reaction. Conversely, the HOC specified on the MATL line refers to the heat of combustion for situations where the material's combustion reaction differs from the dominant reaction. Is it correct to input the heat of combustion in this way?
2. Similarly, for the A/E values, are they analogous to the above case with HOC?
3. I'm attempting to implement finite rate combustion. Do I need to input A/E, NU, and N_S values on the REAC line as mandatory, and is it unnecessary to input HOC?

I think I have some basic questions, and although I've looked at the guide, it's still not clear to me. I'd appreciate your help. Thank you.

**Comment by drjfloyd** [comment]
1. Yes. HOC on REAC defines what happens in the gas phase combustion. If you are using only one gas phase REAC but have different MATL that are pyrolyzing into different fuels, then HOC on MATL allows FDS to adjust the MATL mass loss rate so you get the correct gas phase heat release.
2.  No. A/E on MATL defines the solid phase kinetic. A/E on REAC defines the gas phase kinetics.
3.  If you are doing finite rate reactions , then you must define A/E and NU. N_S you only need to define if it is different from the reaction stoichiometery. HOC is needed if you have not somehow defined reference enthalpies for the SPEC participating in the REAC. Pre-defined FDS species have reference enthalpies. If all of your SPEC in your REAC are pre-defined, you would not need to give an HOC.

Finite rate reactions and predicting pyrolysis are a fairly advanced use of FDS. I would strongly recommend you start with simpler problems until you feel you have learned the basic mechanics of using FDS before jumping into predicting pyrolysis and using finite rate reactions. Once you feel you have reached that point then strongly recommend you focus on the final complex problem in parts, i.e. play with simple cases predicting pyrolysis without finite rate gas phase chemistry or play with cases with finite rate gas phase without predicting pyrolysis. Once you feel you understand the inputs for both individually, then combine together. Too often new users jump straight to complex problems and make multiple input errors for multiple types of input and untangling why a simulation isn't working then becomes far more difficult.

**Reply by MNJOLE** [reply]
I sincerely appreciate your gratitude for the quick and accurate responses. I'm currently trying to organize and clarify the concepts that confuse me as I read the user guide. I'll start with the basic advice you've given and work my way through. Thank you.
